Hunting Queries/ZoomLogs/HighCPURoom.yaml

```diff
@@ -0,0 +1,25 @@
+id: 79cf4646-0959-442f-9707-60fc66eb8145
+name: Zoom room high CPU alerts
+description: |
+  'This hunting query identifies Zoom room systems with high CPU alerts that may be a sign of device compromise.'
+requiredDataConnectors:
+  - connectorId: CustomConnector
+    dataTypes:
+      - ZoomLogs
+tactics:
+  - DefenseEvasion
+  - Persistence
+relevantTechniques:
+  - T1109
+query: |
+
+  let hunt_time = 30d; 
+  ZoomLogs 
+  | where TimeGenerated > ago(hunt_time) 
+  | where Event == "zoomroom.alert" 
+  | extend AlertType = toint(parse_json(RoomEvents).AlertType)  
+  | extend AlertKind = toint(parse_json(RoomEvents).AlertKind) 
+  | where AlertType == 1 and AlertKind == 1 
+  // Uncomment the lines below to analyse event over time
```

Comment:
We should bring thru RoomName and User - | extend RoomName = payload_object_room_name_s, User = payload_object_email_s

Reply:
Also, we still want to bring thru the Entity info just in case someone uses this as an alert - | extend timestamp = TimeGenerated, AccountCustomEntity = User